### PR TITLE
Feature/add new format

### PR DESCRIPTION
**Pull request by kkush**

### CODE REVIEW

File: src/samay.js [hunk 1]
```diff
@@ -110,6 +110,20 @@ function parseDateTime5(strDate) {
 		}
 }
 
+function parseDateTime6(strDate) {
```
Comment (jigar1101) [inline src/samay.js:113]: Why 6?

File: src/samay.js [hunk 2]
```diff
@@ -110,6 +110,20 @@ function parseDateTime5(strDate) {
 		}
 }
```
Comment (jigar1101) [inline src/samay.js:112]: Can we have some comments on the function onto what the function does

File: src/samay.js (hunk 1 above)
Comment (vihangpatel) [reply src/samay.js:113]: There is new parsing that's going to be added.

File: src/samay.js [hunk 3]
```diff
@@ -341,18 +356,19 @@ samay.FORMATS = {
 	'YYYYMMDD' : 'YYYYMMDD',
 	'ddd_DD_MMM_YYYY' : 'ddd, DD MMM, YYYY',
 	'DD_MM_YYYY' : 'DD/MM/YYYY',
-	'MMM_DD_YYYY' : 'MMM DD, YYYY'
+	'MMM_DD_YYYY' : 'MMM DD, YYYY',
+	'YYYY_MM_DD_HH_mm_ss' : 'YYYY-MM-DD HH:mm:ss'
```
Comment (vihangpatel) [inline src/samay.js:360]: See this

File: src/samay.js (hunk 1 above)
Comment (jigar1101) [reply src/samay.js:113]: But then the name should be something about the function. 
```parseDateTime6``` does not make sense

**Pull request merged**